RibonDAO/app
# Feat/new ong card

# Description

- Adds the new ong card

# Necessary changes

- Updates on shared, so the types are correct on the component

## Did you use styled-components?


## Type of change


## How to test

- Open the Causes Screen where the Cards are shown
- Check if the card background image matches de cover_image
- Check if the card title matches the impact_title
